divide_and_conquer/mergesort.py

```diff
@@ -27,13 +27,17 @@ def merge(arr, left, mid, right):
     return arr
 
 
-def mergesort(arr, left, right):
+def mergesort(arr, left=None, right=None):
     """
-    >>> mergesort([3, 2, 1], 0, 2)
+    >>> mergesort([3, 2, 1])
     [1, 2, 3]
-    >>> mergesort([3, 2, 1, 0, 1, 2, 3, 5, 4], 0, 8)
+    >>> mergesort([3, 2, 1, 0, 1, 2, 3, 5, 4])
     [0, 1, 1, 2, 2, 3, 3, 4, 5]
     """
+    if left is None:
```

Comment:
```suggestion
    left = left or 0
```
All other falsy values are as equally dangerous as None.

Reply:
I agree, GitHub is not allowing me to send an extra commit to this pull request, I'll close this in favor of #2444

Reply:
In the upper right of the file, you should see three dots (...).  Click that and select `edit`.

Reply:
I can't, apparently github doesn't allow adding more commits if you remove your folk

![image](https://user-images.githubusercontent.com/35376824/93519200-8d532800-f8f3-11ea-8a6c-0f7a77c1742c.png)